package/MDAnalysis/core/groups.py

```diff
@@ -386,15 +386,26 @@ class _ImmutableBase(object):
     __new__ = object.__new__
 
 
-def check_pbc_and_unwrap(function):
-    """Decorator to raise ValueError when both 'pbc' and 'unwrap' are set to True.
-    """
+def _pbc_to_wrap(function):
+    """Raises deprecation warning if 'pbc' is set and assigns value to 'wrap'"""
+    @functools.wraps(function)
+    def wrapped(group, *args, **kwargs):
+        if kwargs.get('pbc', None) is not None:
+            warnings.warn("The 'pbc' kwarg has been deprecated, "
```

Comment:
add "and will be removed in 3.0"

Reply:
I added comments below that the docs need a deprecation notice in the docs. Maybe you can teach the wrapper to add it automatically?